goenv/version.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // Version of TinyGo.
 // Update this value before release of new version of software.
-const Version = "0.28.0-dev"
+const Version = "0.28.0"
```

Comment:
We probably need to change this to `0.28.1` now?

Reply:
Done.